### DISCUSSION TITLE
NFT showing broken page in Testnet

### DISCUSSION THREAD

**Discussion by natsabari** [spacescan-io/Spacescan-discussion#70] [Support]
The below NFT showing broken page in Test net

https://testnet10.spacescan.io/nft/nft1n4ah23a7rxea9kjdvm43ghn4zl4dnxy7m6kvu7lxz94kd7hpw3dqx2qrma

The data is present though, 
https://api.spacescan.io/nft/info/nft1n4ah23a7rxea9kjdvm43ghn4zl4dnxy7m6kvu7lxz94kd7hpw3dqx2qrma?version=0.1.0&network=txch10&includeActivities=false

**Comment by natsabari** [comment]
@Nithinrcm  Could you please check on this

**Reply by Rigidity** [reply]
Looks like the NFT metadata doesn't follow CHIP-0007 (even though it's marked as such). Perhaps invalid sections of the metadata could be ignored instead.

**Reply by natsabari** [reply]
yea, the data handling should be checked in new code
